code/game/objects/items/ego_weapons/he.dm

```diff
@@ -1747,3 +1747,17 @@
 
 /obj/item/ego_weapon/totalitarianism/get_clamped_volume()
 	return 50
+
+/obj/item/ego_weapon/sunshower //TD
+	name = "sunshower"
+	desc = "I cannot protect you from this rain, but I can guard you from false kindness."
+	icon_state = "sunshower"
+	force = 30
```

Comment:
The dps on this weapon is insane; it should be almost half. 17 or 18 is probably fine. You could also copy some of the code from black swan's weapon to make it a shield of sorts. Balance the stats accordingly either way.